apps/backend/src/user/entities/user.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { IsEmail, IsInt, IsNotEmpty, IsOptional, IsString, Min } from 'class-validator';
+
+export class User {
+  @IsString()
+  @IsNotEmpty()
+  username: string;
+
+  @IsInt()
+  @Min(1)
+  id: number;
+
+  @IsString()
+  @IsNotEmpty()
+  authSchId: string;
+
+  @IsNotEmpty()
+  @IsEmail()
+  email: string;
+
+  @IsString()
+  @IsOptional()
+  profilePicture: string;
+
+  @IsInt()
+  @IsOptional()
+  @Min(0)
+  groups: number;
```

Comment:
We don't need the groups and events list fields. They only exist in prisma's interpretation of the database, but they are not real.